android/src/main/java/com/twiliovoicereactnative/ReactNativeArgumentsSerializer.java

```diff
@@ -142,21 +146,20 @@ public static WritableMap serializeCall(@NonNull final CallRecord callRecord) {
     Objects.requireNonNull(callRecord.getVoiceCall());
 
     // serialize
-    WritableMap callInfo = Arguments.createMap();
-    callInfo.putString(CallInfoUuid, callRecord.getUuid().toString());
-    callInfo.putString(CallInfoSid, callRecord.getVoiceCall().getSid());
-    callInfo.putString(CallInfoFrom, callRecord.getVoiceCall().getFrom());
-    callInfo.putString(CallInfoTo, callRecord.getVoiceCall().getTo());
-    callInfo.putString(CallInfoState, callStateToString(callRecord.getVoiceCall().getState()));
+    WritableMap callInfo = constructJSMap(
+      new Pair<>(CallInfoUuid, callRecord.getUuid().toString()),
+      new Pair<>(CallInfoSid, callRecord.getVoiceCall().getSid()),
+      new Pair<>(CallInfoFrom, callRecord.getVoiceCall().getFrom()),
+      new Pair<>(CallInfoTo, callRecord.getVoiceCall().getTo()),
+      new Pair<>(CallInfoState, callStateToString(callRecord.getVoiceCall().getState())),
+      new Pair<>(CallInfoIsMuted, callRecord.getVoiceCall().isMuted()),
+      new Pair<>(CallInfoIsOnHold, callRecord.getVoiceCall().isOnHold()),
+      new Pair<>(CallInviteInfoCustomParameters, serializeCallInviteCustomParameters(callRecord.getCallInvite())));
```

Comment:
This might be breaking behavior. What does `serializeCallInviteCustomParameters` return? If it returns `null`, then we now have an explicit `null` given to the JS layer instead of an implicit `undefined`.

Reply:
It returns null but if you look at how we generate the map, null data entries get skipped..